src/Symfony/Component/Routing/Matcher/RedirectableUrlMatcherInterface.php

```diff
@@ -25,7 +25,7 @@ interface RedirectableUrlMatcherInterface
      * @param string      $route  The route name that matched
      * @param string|null $scheme The URL scheme (null to keep the current one)
      *
-     * @return array An array of parameters
```

Comment:
would be good to clarify this in the description (returns the parameters for redirection)

Reply:
updated